From 01c69697987f22658778bc911dc2f3bdbc599ce9 Mon Sep 17 00:00:00 2001
Message-Id: <01c69697987f22658778bc911dc2f3bdbc599ce9.1374754301.git.minovotn@redhat.com>
In-Reply-To: <5d75a8513d08b33975bdf5971871c0c977167cd1.1374754301.git.minovotn@redhat.com>
References: <5d75a8513d08b33975bdf5971871c0c977167cd1.1374754301.git.minovotn@redhat.com>
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 24 Jun 2013 07:05:14 +0200
Subject: [PATCH 03/65] qapi: don't convert enum strings to lowercase

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1372057576-26450-4-git-send-email-kraxel@redhat.com>
Patchwork-id: 52120
O-Subject: [RHEL-6.5 qemu-kvm PATCH v2 03/65] qapi: don't convert enum strings to lowercase
Bugzilla: 676568
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

From: Luiz Capitulino <lcapitulino@redhat.com>

Next commit will introduce enum strings in camel case.

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
Reviewed-by: Markus Armbruster <armbru@redhat.com>
(cherry picked from commit ac4ff701d88c4c742b4a53b83eed7ce356535ef8)
---
 scripts/qapi-types.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 scripts/qapi-types.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/qapi-types.py b/scripts/qapi-types.py
index d1b59a0..7e88315 100644
--- a/scripts/qapi-types.py
+++ b/scripts/qapi-types.py
@@ -70,7 +70,7 @@ const char *%(name)s_lookup[] = {
         ret += mcgen('''
     "%(value)s",
 ''',
-                     value=value.lower())
+                     value=value)
 
     ret += mcgen('''
     NULL,
-- 
1.7.11.7

